src/api/java/com/minecolonies/api/util/constant/CitizenConstants.java

```diff
@@ -161,4 +161,27 @@
      * Distance between Barbarian and Citizen to not remove happiness.
      */
     public static final int BARB_DISTANCE_FOR_FREE_DEATH = 21;
+
+
+
+    /**
+     * Tag names for NBT fields
+     */
+    public static final String TAG_NAME = "happiness";
```

Comment:
These are NBT Tag names. There should be an NBTTagConstants class were they belong.